src/lua/lua_url.c

```diff
@@ -370,6 +372,49 @@ lua_url_create (lua_State *L)
 }
 
 static gint
+lua_url_all (lua_State *L)
+{
+	struct rspamd_url *url;
+	rspamd_mempool_t *pool = rspamd_lua_check_mempool (L, 1);
+	const gchar *text,*end;
+	gint length,i;
+	const gchar **pos;
+
+	if (pool == NULL) {
+		lua_pushnil (L);
+	}
+	else {
+		text = luaL_checkstring (L, 2);
+
+		if (text != NULL) {
+			length = strlen(text);
+			*pos=text;
+			end=text+length;
+			lua_newtable(L);
+			while(*pos<=end){
```

Comment:
some style comment: there should be space before opening bracket and spaces before and after `<=`